compiler/lib/src/main/scala/analysis/Semantics/Type.scala

```diff
@@ -345,8 +365,11 @@ object Type {
   }
 
   /** Check for type identity */
-  def areIdentical(t1: Type, t2: Type): Boolean = {
+  def areIdentical(t1Aliased: Type, t2Aliased: Type): Boolean = {
```

Comment:
This change doesn't conform to the spec. The spec says that an alias type is identical to itself, not to its underlying type. See here: https://nasa.github.io/fpp/fpp-spec.html#Type-Checking_Identical-Types.